lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -475,28 +489,29 @@ void ProxyRenderDelegate::_InitRenderDelegate(MSubSceneContainer& container) {
 
 //! \brief  Populate render index with prims coming from scene delegate.
 //! \return True when delegate is ready to draw
-bool ProxyRenderDelegate::_Populate() {
+bool ProxyRenderDelegate::_Populate()
+{
+    TF_VERIFY(_proxyShapeData->ProxyShape());
+    
     if (!_isInitialized())
         return false;
 
-    if (_proxyShapeData->UsdStage() && (!_isPopulated || !_proxyShapeData->IsUsdStageUpToDate() || !_proxyShapeData->IsExcludePrimsUpToDate()) ) {
+    if (_proxyShapeData->UsdStage() && !_isPopulated ) {
         MProfilingScope subProfilingScope(HdVP2RenderDelegate::sProfilerCategory,
             MProfiler::kColorD_L1, "Populate");
 
-        // It might have been already populated, clear it if so.
+        // Remove any excluded prims before populating
```

Comment:
I considered moving this code out of _Populate as well, because it is modifying the render index rather than populating, but I didn't have a strong enough reason to do it so I left it for now.